Makefile

```diff
@@ -259,6 +259,15 @@ generate-mocks: mockery ## Generate the mocks for the project as configured in .
 clean-ui:
 	find ${UI_DIR} -type f -name extension.tar -delete
 
+PHONY: codegen-download-openapi
+codegen-download-openapi: run-backend
+	@echo "Downloading OpenAPI spec from dev server"
+	yarn --cwd ${UI_DIR}  api:download
```

Comment:
I ran the backend in a separate window and executed the `make codegen-download-openapi`. It deleted the whole `ui/src/gen/schema.yaml` file and updated it with the following line:
```
404 page not found
```